### ISSUE TITLE
captum module fails after upgrade to captum v0.3.0

### ISSUE THREAD

**Issue by ababino** [fastai/fastai#3016]
Please confirm you have the latest versions of fastai, fastcore, fastscript, and nbdev prior to reporting a bug (delete one): YES.

**Describe the bug**
The [captum nb](https://github.com/fastai/fastai/blob/master/nbs/73_callback.captum.ipynb) fails after upgrading to captum v0.3.0
I found 3 issues:
1. An import problem.

This one can be fixed by changing 
```python
from captum.insights.features import ImageFeature
```
for 
```python
from captum.insights.attr_vis.features import ImageFeature
```

2. A problem with `NoiseTunnel`. It seem to be related to how `.requiresed_grad_()` works inside `torch.no_grad()` on `Tensor` sub classes. For example, if you run the following code:
```python
x = TensorBase(torch.rand(4,3,16,16))
with torch.no_grad():
    print('x.requires_grad', x.requires_grad)
    y = x.requires_grad_()
    z = x
    print('y.requires_grad', y.requires_grad)
    print('z.requires_grad', z.requires_grad)
```
the output is:
```
x.requires_grad False
y.requires_grad False
z.requires_grad True
```

3. A compatibility problem between fastai `Normalize` and how Captum deals with batched inputs. The behavior that is problematic is the following:
```python
x = TensorImage(torch.rand(3,16,16))
x = Normalize.from_stats(*imagenet)(x)
x.shape
```
now, `x.shape` is `(1,3,16,16)`

**Additional context**

Here is the gist of the analysis with a clearer description of the issues. If you think any of these are good enough to make PR, just let me know, and I'll be happy to make them.

https://gist.github.com/ababino/a675d88ca2826c5eb85b16ebd9c8f446

**Comment by ababino** [comment]
To solve number 2, this patch works well
```python
@patch
def requires_grad_(self:TensorBase, requires_grad=True):
    self.requires_grad = requires_grad
    return self
```
